kie-spring-boot/kie-spring-boot-autoconfiguration/jbpm-spring-boot-autoconfiguration/src/main/java/org/jbpm/springboot/security/SpringSecurityIdentityProvider.java

```diff
@@ -27,16 +29,38 @@
 
 public class SpringSecurityIdentityProvider implements IdentityProvider {
 
+    private Stack<String> contextUsers;
+    
+    public SpringSecurityIdentityProvider() {
+        contextUsers = new Stack<>();
+    }
+    @Override
+    public void setContextIdentity(String userId) {
+        contextUsers.push(userId);
+    }
+
+    @Override
+    public void removeContextIdentity() {
```

Comment:
```suggestion
    public void removeIdentityContext() {
```

Reply:
It is the identity of the context not the other way round.